### PR TITLE
Issue 26/fix script execution

### PR DESCRIPTION
**Pull request by itsmarcy**
Adds `build.sh` that ensures scripts are executed from environment and not PATH. 

- Add `build.sh` to conda.recipe
- Add `samtools` as requirement when running

### CODE REVIEW

File: ngsi_pm_sh/conda.recipe/build.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+mkdir -p $PREFIX/bin
+
+for f in $(ls bin); do
+    (
+        echo "#!$PREFIX/bin/bash"
+        echo "export PATH=$PREFIX/bin:/usr/bin"
+        cat bin/$f
+    ) > $PREFIX/bin/$f
+done
```
Comment (walkerh) [inline ngsi_pm_sh/conda.recipe/build.sh]: Are you missing a newline at the end of this file?

File: ngsi_pm_sh/conda.recipe/meta.yaml [hunk 2]
```diff
@@ -21,4 +20,5 @@ requirements:
   run:
     - bash
     - coreutils
+    - samtools
     - ngsi_pm_py
```
Comment (walkerh) [inline ngsi_pm_sh/conda.recipe/meta.yaml]: Missing newline?

File: ngsi_pm_sh/conda.recipe/build.sh (hunk 1 above)
Comment (walkerh) [reply ngsi_pm_sh/conda.recipe/build.sh]: Including `/usr/bin` suggests that your are not including `coreutils` in your runtime dependencies.

**Review by walkerh — CHANGES_REQUESTED** [review]
Please fix or explain.

File: ngsi_pm_sh/conda.recipe/build.sh (hunk 1 above)
Comment (itsmarcy) [reply ngsi_pm_sh/conda.recipe/build.sh]: `xargs` and `diff` did not work. `coreutils` is in the run time dependencies in the meta.yaml file if thats what you mean.

Not having this caused the commands to return a `command not found` when the path is just `$PREFIX/bin`.

File: ngsi_pm_sh/conda.recipe/meta.yaml (hunk 2 above)
Comment (itsmarcy) [reply ngsi_pm_sh/conda.recipe/meta.yaml]: Added

File: ngsi_pm_sh/conda.recipe/build.sh (hunk 1 above)
Comment (itsmarcy) [reply ngsi_pm_sh/conda.recipe/build.sh]: > Are you missing a newline at the end of this file?

Added

Comment (walkerh) [reply ngsi_pm_sh/conda.recipe/build.sh]: I had no idea that `diff` and `xargs` were missing from the conda ecosystem.

**Review by walkerh — APPROVED** [review]

**Review by jekwatt — APPROVED** [review]
I like that you have added the build script.

**Pull request merged**